### PR TITLE
Miscellaneous accumulated trivia, 202108 edition

### PR DESCRIPTION
**Pull request by nwf**
None of these are expected to have functional implications.

### CODE REVIEW

**Review by davidchisnall — APPROVED** [review]
LGTM, I'll let @mjp41 merge if he's happy with them.

File: src/test/func/memory/memory.cc
Comment (mjp41) [inline src/test/func/memory/memory.cc:259]: The `(void*)` here are due to me being lazy and wanting to print the address in hex, and that is how pointers are printed.  We could fix this with `std::hex << `, or in this case actually using `pointer_offset`.
```suggestion
                  << pointer_offset(p1, size - 1) << std::endl;
```

File: src/test/func/pagemap/pagemap.cc
Comment (mjp41) [inline src/test/func/pagemap/pagemap.cc:43]:
```suggestion
    std::cout << "Location: " << std::hex << address
```

**Review by mjp41 — APPROVED** [review]
LGTM couple of minor bits.

**Review by mjp41 — APPROVED** [review]

**Pull request merged**
